qiskit_sphinx_theme/pytorch_base/static/css/theme.css

```diff
@@ -6,32 +6,8 @@
  * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
  */
 :root {
-  --blue: #007bff;
```

Comment:
These variables were searched for in `theme.css` using my IDE. They can't be used outside of this CSS file.